fwprovider/nodes/resource_oci_image_test.go

```diff
@@ -0,0 +1,283 @@
+//go:build acceptance || all
+
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package nodes_test
+
+import (
+	"context"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+	"github.com/stretchr/testify/require"
+
+	"github.com/bpg/terraform-provider-proxmox/fwprovider/test"
+	"github.com/bpg/terraform-provider-proxmox/proxmox/api"
+	"github.com/bpg/terraform-provider-proxmox/proxmox/helpers/ptr"
+	"github.com/bpg/terraform-provider-proxmox/proxmox/nodes/storage"
+)
+
+const (
+	testOCIImage = "docker.io/library/hello-world:latest"
+)
+
+func TestAccResourceOCIImage(t *testing.T) {
+	te := test.InitEnvironment(t)
+
+	te.AddTemplateVars(map[string]interface{}{
+		"OCIImage": testOCIImage,
+	})
+
+	tests := []struct {
+		name  string
+		steps []resource.TestStep
+	}{
+		{"missing reference", []resource.TestStep{{
+			Config: te.RenderConfig(`
+				resource "proxmox_oci_image" "test_image" {
```

Comment:
`proxmox_oci_image` won't work, should be `proxmox_virtual_environment_oci_image`

Reply:
<img width="800" height="503" alt="Capture d’écran du 2025-12-05 14-58-44" src="https://github.com/user-attachments/assets/7aee2c2a-8605-488c-b25d-06ea9cf4e50a" />